object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -1,46 +1,61 @@
 = Define Object Stores Examples - Mule 4
 :page-aliases: connectors::object-store/object-store-to-define-a-new-os.adoc
 
-You can define an Object Store globally in the app so it can be referenced by name and shared between multiple components, or you can create an Object Store that only for a particular component.
+You can define an Object Store globally in the app so it can be referenced by name and shared between multiple components, or you can create an Object Store for one particular component.
```

Comment:
```suggestion
You can define an Object Store globally in the app so that it can be referenced by name and shared between multiple components, or you can create an Object Store for one particular component.
```